docker/dockerfiles/artifact-ubuntu.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This docker file build the Starrocks artifacts fe & be and package them into a busybox-based image
+# Please run this command from the git repo root directory to build:
+# DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/artifact-ubuntu.Dockerfile -t starrocks/artifact-ubuntu:tag .
+
+ARG builder=ghcr.io/dengliu/starrocks/dev-env-ubuntu:latest
+
+FROM ${builder} as fe-builder
+# clean and build Frontend and Spark Dpp application
+COPY . /build/starrocks
+WORKDIR /build/starrocks
+RUN MAVEN_OPTS='-Dmaven.artifact.threads=128' ./build.sh --fe --clean
```

Comment:
suggest add build arg to not `--clean` every time, so it will be much friendly for developers to repeat change-code-and-build